File: src/main/resources/front_ts_vue/src/entities/image/model/Image.ts
export default class Image {
  private _id: number;
  private _path: string;

  constructor(id: number, path: string) {
    this._id = id;
    this._path = path;
  }

  get id(): number {
    return this._id;
  }

  set id(value: number) {
    this._id = value;
  }

  get path(): string {
    return this._path;
  }

  set path(value: string) {
    this._path = value;
  }
}

File: src/main/java/com/faketri/market/entity/image/model/Image.java
package com.faketri.market.entity.image.model;

import jakarta.persistence.*;
import lombok.Getter;
import lombok.Setter;
import lombok.ToString;
import org.hibernate.annotations.JdbcTypeCode;
import org.hibernate.type.SqlTypes;

/**
 * The type Image.
 *
 * @author Dmitriy Faketri
 */
@Getter
@Setter
@ToString
@Entity
public class Image {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    @Column(name = "id", nullable = false)
    @JdbcTypeCode(SqlTypes.BIGINT)
    private Long id;

    @Column
    private String path;

    /**
     * Instantiates a new Image.
     */
    public Image() {
    }

    /**
     * Instantiates a new Image.
     *
     * @param id   the id
     * @param path the path
     */
    public Image(Long id, String path) {
        this.id = id;
        this.path = path;
    }

}